scalyr_agent/configuration.py

```diff
@@ -1709,6 +1716,14 @@ def __verify_main_config_and_apply_defaults(
             apply_defaults,
             env_aware=True,
         )
+        self.__verify_or_set_optional_bool(
+            config,
+            "global_monitor_sample_interval_enable_jitter",
+            False,
```

Comment:
Maybe we set this to `True` by default?  It seems to me that this falls into that category of "turning it on shouldn't negatively impact anyone" and "if you need this, it is going to be hard to find this option".

Reply:
Yeah, I agree.

I may just wait till the next release so we have a chance to run it on staging for a while just to be sure.